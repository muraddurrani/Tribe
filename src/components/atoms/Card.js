import React from 'react'
import { View, StyleSheet } from 'react-native'
import colours from '../../styles/colours'

function Card({ style, children }) {
  return (
    <View style = {{...styles.view, ...style}}>
      {children}
    </View>

  )
}

const styles = StyleSheet.create({
  view: {
    backgroundColor: colours.gray0,
    borderRadius: 15,
    elevation: 3
  }
})

export default Card
