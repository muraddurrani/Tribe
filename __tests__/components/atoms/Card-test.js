import React from 'react';
import renderer from 'react-test-renderer';
import Card from '../../../src/components/atoms/Card'

jest.useFakeTimers()

it('Card renders correctly', () => {
  const tree = renderer
    .create(<Card style = {{alignItems: 'center', width: '90%'}}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});