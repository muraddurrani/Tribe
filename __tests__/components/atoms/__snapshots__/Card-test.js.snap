// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fdfdfd",
      "borderRadius": 15,
      "elevation": 3,
      "width": "90%",
    }
  }
/>
`;
